officers.json

```diff
@@ -41,6 +41,20 @@
     "term_end": "SPRING_2019",
     "graduating_semester": "SPRING_2021"
   },
+  {
+    "name": "Joshua Castor",
+    "position": "Technical Officer",
```

Comment:
New positions will probably need to also be changed in `server.py`, looking at how it's implemented in #31

Reply:
Ok I'll make a change for that and get a PR for that